portugues_pt/mundo/startup/startup.mdown

```diff
@@ -0,0 +1 @@
+Começava uma nova empresa com o novo produto revolucionário do mercado
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.